.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Tas ir nepieciešams? Nekad neredzeju agrāk lai to šeit definetu.
Just asking.

Reply:
Es pats isti nezinu , tas tika autogeneratots kad taisijas rspec mapite ar tiem failiem, bet manliekas ka bez sita man kkas negaja terminali censoties palaist rspec